modules/services/mako.nix

```
@@ -4,13 +4,13 @@ with lib;
 
 let
 
-  cfg = config.programs.mako;
+  cfg = config.services.mako;
 
 in {
   meta.maintainers = [ maintainers.onny ];
 
   options = {
-    programs.mako = {
+    services.mako = {
       enable = mkEnableOption ''
         Mako, lightweight notification daemon for Wayland
       '';
```

Please, make the package configurable

```suggestion
      enable = mkEnableOption ''
        Mako, lightweight notification daemon for Wayland
      '';
      
      package = mkOption {
        type = types.package;
        default = pkgs.mako;
        defaultText = "pkgs.mako";
        description = ''
          mako derivation to use.
        '';
      };
```